### ISSUE TITLE
URL stopped working

### ISSUE THREAD

**Issue by Richard-L** [jxnblk/colorable#38]
Thank you so much for this wonderful little tool. I love the way you've designed it.

As of late, I notice that linking someone URLs with colour values seem to 404. For example: https://colorable.jxnblk.com/3d93f7/292929
![image](https://user-images.githubusercontent.com/4934209/181225015-c914ed2c-12d4-4df0-b768-70aeffa101d5.png)

If it is at all in your power to fix this, I would appreciate it a lot.
Best to you,
Richard

**Comment by adrian5** [comment]
Right, I also noticed this when using it just for myself. As the site constantly adjusts the URL on the fly, revisiting a Colorable tab at a later time will likewise fail.
